src/main/java/org/assertj/core/api/AbstractIterableAssert.java

```diff
@@ -924,6 +925,45 @@ public S usingDefaultElementComparator() {
 	return new ListAssert<>(result);
   }
 
+    /**
+     * Extract the Iterable values from Iterable's elements under test by applying an Iterable extracting function based
```

Comment:
I feel we could simplify this explanation by avoiding to mention the extracting function as it is an implementation detail. Maybe something like:
```
Extract from Iterable's elements the Iterable values corresponding to the given property/field name and concatenate them into a single list becomes a new object under test.
```